pkgs/servers/nextcloud/default.nix

```diff
@@ -42,18 +42,6 @@ let
     };
   };
 in {
-  nextcloud25 = throw ''
-    Nextcloud v25 has been removed from `nixpkgs` as the support for is dropped
-    by upstream in 2023-10. Please upgrade to at least Nextcloud v26 by declaring
-
-        services.nextcloud.package = pkgs.nextcloud26;
-
-    in your NixOS config.
-
-    WARNING: if you were on Nextcloud 24 you have to upgrade to Nextcloud 25
-    first on 23.05 because Nextcloud doesn't support upgrades across multiple major versions!
-  '';
-
```

Comment:
This is still fairly recent. Can we not remove this?

Reply:
Not sure what you mean. I just moved this part around while keeping the same error message, since I know close to nothing about all of these packages :laughing:

Reply:
Ah just saw https://github.com/NixOS/nixpkgs/pull/276112#discussion_r1435352321 haha